### DISCUSSION TITLE
Second maintainer

### DISCUSSION THREAD

**Discussion by nojaf** [giraffe-fsharp/Giraffe#583] [General]
Hi @64J0,

Great to see you stepping up with Giraffe here.
What is the status of having a second maintainer for this project?
Without more maintainers, I feel like we just switched the problem from Dustin to you.

**Comment by dustinmoris** [comment]
> What is the status of having a second maintainer for this project?

No second maintainer has volunteered yet if I'm not mistaken.

**Reply by 64J0** [reply]
For now, @TheAngryByrd helps with PR reviews sometimes

**Reply by nojaf** [reply]
Regarding Jimmy, my memory tells me he's not keen on taking on this responsibility.

**Reply by TheAngryByrd** [reply]
Yeah I'll chip in when I can but I already have my hands full of other projects. Definitely feel free to ping me if you want help with something specific though.

**Comment by 64J0** [comment]
I was considering from [this issue](https://github.com/fsprojects/.github/issues/49#issuecomment-1959397550) that both you (@nojaf) and @dsyme would be maintainers along with me.

**Reply by nojaf** [reply]
No, think of us as a support system for administrative hurdles. For instance, if communication breaks down and releases come to a halt, that's when we step in. Of course, we hope it never comes to that.

**Comment by nojaf** [comment]
I firmly think you need someone else familiar with the codebase who can respond to issues, manage pull requests, and handle releases. It's not about fixing every issue with Giraffe yourself, but rather making sure contributors can navigate and contribute effectively.

**Comment by dbrattli** [comment]
I would be interested in helping out as a secondary maintainer. I already have some experience with the codebase ref [Giraffe.Pipelines](https://github.com/dbrattli/Giraffe.Pipelines) for using `|>` instead of `>=>` and [Fable.Giraffe](https://github.com/dbrattli/Fable.Giraffe) where I transpile Giraffe to Python using Fable.

**Reply by nojaf** [reply]
Thanks Dag! This is great news, maybe @64J0 and you can jump on a short call sometime? Get acquainted and align with your expectations.

**Reply by dbrattli** [reply]
Thanks, I've added you both to a private group on F# Slack where we can continue the discussion and arrange with a call
